modules/gdnative/gdnative.cpp

```diff
@@ -847,6 +847,16 @@ bool GDNativeInstance::set(const StringName &p_name, const Variant &p_value) {
 		script->script_data->properties[p_name].setter.set_func((godot_object *)owner, script->script_data->properties[p_name].setter.method_data, userdata, *(godot_variant *)&p_value);
 		return true;
 	}
+
+	Map<StringName, GDNativeScriptData::Method>::Element *E =  script->script_data->methods.find("_set");
```

Comment:
It's fine. This is an element of that Map, either it's null, then you ignore it or you operate on the contents, it's not a big deal, it's done everywhere in every loop over Maps and other containers, it's fine.